src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,7 +5,33 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
+    private int loadCapacity;
+    private String typeOfFuel;
+
     public Truck() {
+        super("Default name", "Default color");
+    }
+
+    public Truck(String name, String color, int loadCapacity, String typeOfFuel) {
+        super(name, color);
+        this.loadCapacity = loadCapacity;
+        this.typeOfFuel = typeOfFuel;
+    }
+
+    public int getLoadCapacity() {
+        return loadCapacity;
+    }
+
+    public void setLoadCapacity(int loadCapacity) {
+        this.loadCapacity = loadCapacity;
+    }
+
+    public String getTypeOfFuel() {
+        return typeOfFuel;
+    }
+
+    public void setLoadCapacity(String typeOfFuel) {
+        this.typeOfFuel = typeOfFuel;
```

Comment:
The method name `setLoadCapacity` is incorrect for setting the `typeOfFuel` field. It should be `setTypeOfFuel` to match the field it is intended to modify.